src/ErrorOr.cs

```diff
@@ -21,6 +21,11 @@ public record struct ErrorOr<TValue> : IErrorOr
     /// </summary>
     public bool IsError { get; }
 
+    /// <summary>
+    /// Gets a value indicating whether the state is success (error didn't happen).
+    /// </summary>
+    public bool IsSuccess { get => !IsError; }
```

Comment:
I think this is similar to duplication, why just not do IsError

Reply:
I agree. I prefer having only the `IsSuccess` property. For some reason I feel it increases the cognitive load across the project and may push people to write code like this:

```csharp
if (result.IsSuccess)
{
    // is success logic - often multiple LOC
}

return result.Errors;
```

Instead of this:
```csharp
if (result.IsError)
{
    return result.Errors;
}

// is success logic - often multiple LOC
```